mcsema/Arch/X86/Semantics/String.cpp

```diff
@@ -859,4 +864,7 @@ void String_populateDispatchMap(DispatchMap &m) {
   m[llvm::X86::REPNE_CMPSW_64] = translate_CMPS16;
   m[llvm::X86::REPNE_CMPSD_64] = translate_CMPS32;
   m[llvm::X86::REPNE_CMPSQ_64] = translate_CMPS64;
+
+  m[llvm::X86::REP_PREFIX] = translate_REP_prefix;
```

Comment:
Question: in this version of mcsema, how do we detect that an instruction is REP-prefixed? Does everything have its own rep-enum (e.g. REP_MOVSD_32) or do some emit a REP_PREFIX followed by the actual instruction?